cypress/fixtures/docs/validations.md

```diff
@@ -421,6 +422,81 @@ The 'schemaOnly` validation type allows to skip content validation and stops at
 
 ```
 
+## screenshotValidation
+To perform screenshot validation, the screenshot validation object should be added to the validation objects.
+
+- The screenshot validation will happen only when the `enableScreenshots` environment variable is set to `true`.
+- FCA look for screenshotValidation status in below format and based on the status, it will decide the overall status of the screenshot validation.
+  ```javascript
+  {
+    status: "pass/fail",
+    validations: [
+      {status: "pass/fail"},
+      {status: "pass/fail"}
+    ]
+  }
+  ```
+- Actual validation can be added in cofigModule override function. Refer [here](/Docs/Request_Overrides.md#screenshot) for more details on `screenshot` override.
+
+### Validation object format: 
+
+```json
+{
+  "data": [
+    {
+      "type": "screenshotValidation",
+      "validations": []
+    }
+  ]
+}
+```
+
+### Params:
+
+| **Param**   | **Type** | **Description**                                                            |
+| ----------- | -------- | -------------------------------------------------------------------------- |
+| data        | array    | An array that holds the entire set of validation objects.                  |
+| type        | string   | The value that indicates the type of validation.                           |
+| validations | array    | The array that holds all the data for validation, like value, format, etc. |
+
+### Examples:
+**Example 1:** Below validation objects consist of OCR and image validation. The OCR validation is to validate the text `tv+` and the image validation is to validate the image with label `auth` with confidence 60 against the screenshot response.
+
+```json
+{
+  "data": [
+    {
+      "type": "screenshotValidation",
+      "validations": [
+        {
+            "type": "ocr",
+            "text": "tv+",
+            "regex": "tv"
```

Comment:
Should the regex format be given here ? or we have a file for the regex formats ?

Reply:
regex format given here only